streams/src/main/java/org/apache/kafka/streams/state/RocksDBConfigSetter.java

```diff
@@ -37,4 +37,20 @@
      * @param configs       the configuration supplied to {@link org.apache.kafka.streams.StreamsConfig}
      */
     void setConfig(final String storeName, final Options options, final Map<String, Object> configs);
+
+    /**
+     * This method should close any user-constructed objects that inherit from {@code org.rocksdb.RocksObject}
+     *
+     * Any such object created with @{code new} in {@link #setConfig} should have {@code close} called on it here to
+     * avoid leaking off-heap memory. Objects to be closed can be saved by the user or retrieved back from
+     * {@code options} using its getter methods.
+     *
+     * Example objects needing to be closed include {@code org.rocksdb.Filter} and {@code org.rocksdb.Cache}.
+     *
+     * @param storeName     the name of the store being configured
+     * @param options       the Rocks DB options
```

Comment:
nit `RocksDB` without space

Reply:
whoops, that was just copied from above. I'll fix it in #setConfig too